Source/WebKit/Shared/WebHitTestResultData.cpp

```diff
@@ -109,124 +109,44 @@ WebHitTestResultData::WebHitTestResultData(const HitTestResult& hitTestResult, b
         }
     }
 }
+#if PLATFORM(MAC)
+WebHitTestResultData::WebHitTestResultData(const String& absoluteImageURL, const String& absolutePDFURL, const String& absoluteLinkURL, const String& absoluteMediaURL, const String& linkLabel, const String& linkTitle, const String& linkSuggestedFilename, bool isContentEditable, const WebCore::IntRect& elementBoundingBox, const WebKit::WebHitTestResultData::IsScrollbar& isScrollbar, bool isSelected, bool isTextNode, bool isOverTextInsideFormControlElement, bool isDownloadableMedia, const String& lookupText, const String& toolTipText, const String& imageText, const std::optional<WebKit::SharedMemory::Handle>& imageHandle, const RefPtr<WebKit::ShareableBitmap>& imageBitmap, const String& sourceImageMIMEType, const WebHitTestResultPlatformData& platformData, const WebCore::DictionaryPopupInfo& dictionaryPopupInfo, const RefPtr<WebCore::TextIndicator>& linkTextIndicator)
```

Comment:
This could just have PLATFORM(MAC) around the one parameter like in the header.  Less redundancy.